app/helpers/field_validation_helper.rb

```diff
@@ -76,6 +76,10 @@ def validate_date_of_birth(year, month, day, field)
     unless(invalid_fields != [] || DateTime.new(year.to_i, month.to_i, day.to_i) < Time.zone.now)
       invalid_fields << { field: "#{field}-day", text: t("coronavirus_form.errors.date_order") }
     end
+    # Check for date being more than 150 years ago
+    unless(invalid_fields != [] || DateTime.new(year.to_i, month.to_i, day.to_i) >= Time.zone.now - 150.years)
```

Comment:
Why the check for invalid_fields not being blank? I realise that it's the same as the other checks, but I'm curious.

Reply:
You could end up with a list of validation errors with (potentially) all the same error message if every validation were to run

Reply:
Also, `Time.zone.now - 150.years` could be written as `150.years.ago`

Reply:
Thanks, have updated to use `150.years.ago`

Reply:
Perhaps rather than adding `invalid_fields != []` to the beginning of each check, they can all be wrapped in:
```
if invalid_fields.blank?
   date_off_birth = DateTime.new(year.to_i, month.to_i, day.to_i)
   
  if date_off_birth in the future
  if date_off_birth >= 150.years.ago
end
```
etc

Reply:
Perhaps we should consider this in another refactor.  I'm sure there was a design reason why we're showing a single error message, rather than doing all the validations at once.